### PR TITLE
updated cloneRepository function

### PR DESCRIPTION
**Pull request by dixyushi**

### CODE REVIEW

File: documentation/Functions.md [hunk 1]
```diff
@@ -252,15 +252,24 @@ createFolder("directoryPath/subDirectory")
...
 #### parameter 
-1. Path into which the repository is to be cloned, relative to workspace.
-2. Git repository URL
+You need two parameter for this function.
+
+Example: cloneRepository(1st-parameter, 2nd-parameter)
+
+1st-parameter: It is path into which the repository is to be cloned, relative to workspace. It is of type String.
+
+2nd-parameter: It is the git repository URL. It is of type String.
```
Comment (GuentherJulian) [inline documentation/Functions.md:263]: Git with leading capital letter

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: documentation/Functions.md [hunk 2]
```diff
@@ -252,15 +252,24 @@ createFolder("directoryPath/subDirectory")
 ***
 
 ### cloneRepository <a name="cloneRepository"></a>
+This function is used when you want to clone a repository into a particular repository.
```
[the remark this answers is not in the document]
Comment (dixyushi) [reply documentation/Functions.md:255]: changed

File: documentation/Functions.md (hunk 1 above)
Comment (dixyushi) [reply documentation/Functions.md:263]: done

File: documentation/Functions.md [hunk 3]
```diff
@@ -252,15 +252,32 @@ createFolder("directoryPath/subDirectory")
...
-cloneRepository(devonfw-tutorials", "https://github.com/devonfw-tutorials/tutorial-compiler.git")
-Repository will be cloned into a newly created subdirectory devonfw-tutorials.
+2. cloneRepository("devonfw-tutorials", "https://github.com/devonfw-tutorials/tutorial-compiler.git")
+Repository will be cloned into a newly created subdirectory "devonfw-tutorials".
+
+Note: 
+
+1. No scripts will run in the background. 
+
+2. The command for execution will be generated by katacoda runner, so user will have to execute this command manually.
```
Comment (GuentherJulian) [inline documentation/Functions.md:279]: Katacoda

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: documentation/Functions.md (hunk 3 above)
Comment (dixyushi) [reply documentation/Functions.md:279]: Changed!!

**Pull request merged**